client/components/Profile/EditableAvatar.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { ReactElement, useState, ChangeEvent } from "react";
+import Avatar from "@material-ui/core/Avatar";
+import AddAPhotoIcon from "@material-ui/icons/AddAPhoto";
+import { AxiosPromise } from "axios";
+import useStyles from "./Profile.styles";
+import API from "../../util/api";
+
+interface Props {
+  picture?: string;
+}
+
+interface ApiResponse {
+  id: string;
+}
+
+export const sendPictureData = (picture: File): AxiosPromise<ApiResponse> => {
+  const data = new FormData();
+  console.log("AGNEUGEGEGE", picture);
+  data.append("image", picture, picture.name);
```

Comment:
you don't need to add `picture.name`
```suggestion
  data.append("image", picture);
```